docs/img/crux-node-1.svg

```diff
@@ -485,16 +485,11 @@
        id="text2061-84-7-0-4-5-8-3-9-2-1-4"
        inkscape:export-xdpi="60"
        inkscape:export-ydpi="60"><tspan
-         sodipodi:role="line"
-         x="149.2623"
-         y="31.105722"
-         style="font-weight:bold;font-size:7.76111126px;line-height:1.25;text-align:start;text-anchor:start;stroke-width:0.18165512"
-         id="tspan2155-0-6-3-7-0-5-4-7-4-4-3">Put/</tspan><tspan
          sodipodi:role="line"
          x="149.2623"
          y="40.80711"
          style="font-weight:bold;font-size:7.76111126px;line-height:1.25;text-align:start;text-anchor:start;stroke-width:0.18165512"
-         id="tspan7694-9-0">CAS/</tspan><tspan
+         id="tspan7694-9-0">Put/</tspan><tspan
```

Comment:
Hard to tell without looking at the SVG, but should this maybe be "Match"?

Reply:
if memory serves, it was originally 3 lines, put, cas, delete - I replaced cas with put and removed the original put, because they were labels for a line below